app/static/src/app/store/model/mutations.ts

```diff
@@ -26,10 +31,15 @@ const runRequired = (state: ModelState) => {
 export const mutations: MutationTree<ModelState> = {
     [ModelMutation.SetOdinRunner](state: ModelState, payload: string) {
         state.odinRunner = evaluateScript<OdinRunner>(payload);
+        state.odinRunnerError = null;
     },
 
     [ModelMutation.SetOdinResponse](state: ModelState, payload: OdinModelResponse) {
         state.odinModelResponse = payload;
+        state.odinModelResponseError = null;
```

Comment:
Could we call this `odinModelCodeError`? - I known it comes out of the response, but odinModelResponseError makes it sounds like an api error.